src/main/java/Operator.java

```diff
@@ -0,0 +1,239 @@
+
+import java.util.*;
+
+public class Operator {
+    // Operator handles the user input and output
+    private Scanner scanner;
+    private TaskList taskList;
+
+    public Operator() {
+        this.scanner = new Scanner(System.in);
+        this.taskList = new TaskList();
+    }
+
+    // Entry point of the bot
+    public void goLive() throws BotException {
```

Comment:
Perhaps you can consider breaking the goLive() method into more simpler methods, maybe each could handle a specific command?